data/helpers.d/package

```diff
@@ -117,8 +117,13 @@ ynh_package_install_from_equivs () {
 #
 # usage: ynh_install_app_dependencies dep [dep [...]]
 # | arg: dep - the package name to install in dependence
+#   You can give a choice between some package with this syntax : "dep1|dep2"
+#   Example : ynh_install_app_dependencies dep1 dep2 "dep3|dep4|dep5"
+#   This mean in the dependence tree : dep1 & dep2 & (dep3 | dep4 | dep5)
 ynh_install_app_dependencies () {
     local dependencies=$@
+    local dependencies=${dependencies// /, }
```

Comment:
What about if we put 2 comas ?
see https://github.com/YunoHost-Apps/grafana_ynh/blob/e5a150e077eb9120d9e41a3bad6404c63d33f2fe/scripts/_common.sh#L60

Reply:
Yes, it's possible but in this case we change the synatax of the actuall helper. My purpose is juste add the choice possibility with the same synatx.

Actually we have this synaxe : `ynh_install_app_dependencies dep1 dep2 dep 3`
And with my purpose this will be that : `ynh_install_app_dependencies dep1 dep2 "dep3|dep4|dep5"` witch mean : dep1 & dep2 & (dep3 | dep4 | dep5)

Reply:
I think the question here was to know what happens if the users already had `,` in the arguments, like grafana_ynh has : `ynh_install_app_dependencies influxdb, grafana`, which means `$dependencies` will end up containing `influxdb,, grafana`, but thinking about it, that shouldn't be any trouble since this line was actually done later in the script. So I'm guessing those debian control files things are robust against typos like having `,,` :+1:

Reply:
Ah, ok I understand, but I've just moved the replacement, but it was already here. (see : https://github.com/YunoHost/yunohost/pull/381/files#diff-48ee89702d6221dfe3ea24923b0f8f9eR143). I think the debian script is really exigent about the syntax, so two comma won't probably be accepted.